### PR TITLE
Migrate charging station service abac

### PR DESCRIPTION
**Pull request by r0mux**

### CODE REVIEW

File: src/app/pages/charging-stations/list/charging-stations-list-table-data-source.ts [hunk 1]
```diff
@@ -116,105 +137,93 @@ export class ChargingStationsListTableDataSource extends TableDataSource<Chargin
...
-          headerClass: 'd-none d-xl-table-cell col-20p',
-        },
-        {
-          id: 'siteArea.name',
-          name: 'site_areas.title',
-          class: 'd-none d-xl-table-cell col-20p',
-          headerClass: 'd-none d-xl-table-cell col-20p',
-        },
-      );
+    const columns: TableColumnDef[] = [];
```
Comment (LucasBrazi06) [inline src/app/pages/charging-stations/list/charging-stations-list-table-data-source.ts:140]: You can return right away the column def, no need an extra var.

**Review by LucasBrazi06 — COMMENTED** [review]
Just one global remark before I review this PR.
Check that no 'if' statement is used in method buildTableColumnDefs() in Data Source. Use the visible property.
It seems already to be the case in this PR but must be checked in all Data Sources (maybe in another PR)
Thanks!

**Comment by Ramzay** [comment]
> Just one global remark before I review this PR. Check that no 'if' statement is used in method buildTableColumnDefs() in Data Source. Use the visible property. It seems already to be the case in this PR but must be checked in all Data Sources (maybe in another PR) Thanks!

Yes it'

**Review by LucasBrazi06 — APPROVED** [review]
GUTGUTGUT!!!

**Pull request merged**